geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/RegionProvider.java

```diff
@@ -345,6 +345,14 @@ public RedisKeyCommands getKeyCommands() {
     return (Set<DistributedMember>) (Set<?>) partitionedRegion.getRegionAdvisor().adviseDataStore();
   }
 
+  /**
+   * A means to consistently order multiple keys for locking to avoid typical deadlock situations.
+   * Note that the list of keys is sorted in place.
+   */
+  public void orderForLocking(List<RedisKey> keys) {
+    keys.sort(stripedCoordinator::compareStripes);
+  }
```

Comment:
You don't need to use this anymore since locking (and sorting) is automatically handled in the `LockingStripedCoordinator` now.